packages/run-protocol/src/contractSupport.js

```diff
@@ -79,3 +80,26 @@ export const checkDebtLimit = (debtLimit, totalDebt, toMint) => {
     )}`,
   );
 };
+
+/**
+ * @template T
+ * @param {ERef<StorageNode>} [storageNode]
+ * @param {ERef<Marshaller>} [marshaller]
+ * @returns {MetricsSubscriptionKit<T>}
+ */
+export const makeMetricsSubscriptionKit = (storageNode, marshaller) => {
```

Comment:
Do we really need this wrapper?  It changes the API surface in an arbitrary way that I find distracting.

What's wrong with open-coding the call `makeStoredPublisherKit` and making the caller more explicitly request a `'metrics'` child node?

Reply:
> Do we really need this wrapper?

Of course we could repeat the code everywhere instead.

> What's wrong with open-coding the call

Nothing wrong with that. This is to reify a pattern that is used throughout the run-protocol contracts so that `metricsPublication` in the result is always the `metrics` node.

It's in contractSupport which has other convenience methods.

Reply:
Okay, that's a good enough argument.  I just wanted to make sure that the helper paid its way.